patch/dpdk-stable-17.11.6/0003-pdump-enable-and-change-dpdk-pdump-tool-for-DPVS.patch

```diff
@@ -370,11 +372,14 @@ diff -uparN /home/limiao/dpvs/dpdk-stable-17.11.6/lib/librte_pdump/rte_pdump.c d
 +                af = AF_INET;
 +                s_addr.in.s_addr = arp->arp_data.arp_sip;
 +                d_addr.in.s_addr = arp->arp_data.arp_tip;
++                proto = IPPROTO_ICMP; 
```

Comment:
I don't think `proto = IPPROTO_ICMP` is needed here. ARP packets cannot be captured in this case.